README.md

```diff
@@ -9,8 +9,9 @@ This package provides `Control.Effect.Exception`, a module that wraps the [`Cont
 Please be aware that injudicious use of these functions may provoke surprising interactions with carriers that thread a monadic state as a parameter, à la the `Control.Carrier.State` types provided by `fused-effects`. For example, a function like `finally`, which does not thread any state from its body to its handler block, may discard state writes in cleanup handlers:
 
 ```haskell
-λ runM (runResource (runState 'a' (modify (succ @Char) `finally` modify (succ . succ @Char))))
-('b', ())
+λ run (runState 'a' ((throwIO (userError "urk") `finally` put @Char 'z')
+    `catch` (\(_ :: IOException) -> pure ())))
+('a', ())
```

Comment:
@patrickt: This is incorrect:

1. We can’t use `run` here because `run` is the handler for `Identity`. If you want to run in `IO`, which I surmise you do since you’re using `throwIO` and `finally` and all, then you don’t use a handler for it at all.

2. Multiline ghci doesn’t work like this.

I think we want to do this instead:

```haskell
λ runState 'a' ((throwIO (userError "urk") `finally` put @Char 'z') `catch` (\(_ :: IOException) -> pure ()))
```